iaso/api/org_unit_change_request_configurations/views_mobile.py

```diff
@@ -70,7 +79,12 @@ def list(self, request: Request, *args, **kwargs) -> Response:
 
         """
         app_id = AppIdSerializer(data=self.request.query_params).get_app_id(raise_exception=True)
+        include_creation = IncludeCreationSerializer(data=self.request.query_params).get_include_creation(
+            raise_exception=True
+        )
```

Comment:
This could be handled by a custom `filterset_class` instead of a brand new `IncludeCreationSerializer `.

IMO it would be less verbose and just as effective.

Reply:
You can see an example of what I mean here:

- https://github.com/BLSQ/iaso/blob/413f5c1ef6887bf351bc8f65a9074becb3128c74/iaso/api/org_unit_change_request_configurations/views.py#L39
- https://github.com/BLSQ/iaso/blob/413f5c1ef6887bf351bc8f65a9074becb3128c74/iaso/api/org_unit_change_request_configurations/filters.py#L9

Don't bother if you don't feel comfortable, this could be handled later.